cesium_app/tests/frontend/test_predict.py

```diff
@@ -204,12 +204,30 @@ def test_download_prediction_csv_class(driver):
             os.remove('/tmp/cesium_prediction_results.csv')
 
 
+def test_download_prediction_csv_class_unlabeled(driver):
+    driver.get('/')
+    with create_test_project() as p, create_test_dataset(p, label_type=None) as ds,\
+         create_test_featureset(p) as fs,\
+         create_test_featureset(p, label_type=None) as unlabeled_fs,\
+         create_test_model(fs, model_type='LinearSGDClassifier') as m,\
+         create_test_prediction(ds, unlabeled_fs, m):
+        _click_download(p.id, driver)
```

Comment:
If we can change `_click_download` to use the `tmpdir` fixture that `pytest` provides then we don't have to handle the `finally: <remove file>` part